client/src/pages/signUp.js

```diff
@@ -0,0 +1,157 @@
+import React, { useState } from 'react';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+import Link from '@material-ui/core/Link';
+import Paper from '@material-ui/core/Paper';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import Icon from '@material-ui/core/Icon';
+import { makeStyles } from '@material-ui/core/styles';
+
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    height: '100vh',
+  },
+  image: {
+	background: 'linear-gradient(to bottom, #A83AFD, #8225FB)',
+    backgroundPosition: 'center',
+	display: 'flex',
+    flexDirection: 'column',
+	alignItems: 'center',
+  },
+  paper: {
+    margin: theme.spacing(10),
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'flex-start',
+  },
+  form: {
+	width: '100%',
+	marginTop: theme.spacing(1),
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2),
+  },
+  icon: {
+    marginTop: theme.spacing(20),
+	marginBottom: theme.spacing(5),
+	fontSize : 100,
+  },
+}));
+
+export default function SignUp() {
+  const classes = useStyles();
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+  const [pword, setPword] = useState("");
+  const [cpassword, setCpassword] = useState("");
```

Comment:
use proper camel case here, and better names here (like setConfirmPassword, and confirmPassword) - this clean coding is very important when working in a team